csharp/test/Drivers/Apache/Spark/DriverTests.cs

```diff
@@ -309,6 +341,39 @@ public void CanGetObjectsAll()
 
             Assert.True(columns != null, "Columns cannot be null");
             Assert.Equal(TestConfiguration.Metadata.ExpectedColumnCount, columns.Count);
+
+            for (int i = 0; i < columns.Count; i++)
+            {
+                // Verify column metadata is returned/consistent.
+                AdbcColumn column = columns[i];
+                Assert.Equal(i + 1, column.OrdinalPosition);
+                Assert.NotNull(column.Name);
```

Comment:
redundant, as null is checked in the following line: `Assert.False(string.IsNullOrEmpty(column.Name));`

Reply:
Removed